std/didexchange/v0/models.go

```diff
@@ -88,7 +88,8 @@ func (c *Connection) UnmarshalJSON(b []byte) (err error) {
 	defer err2.Handle(&err, "unmarshal connection")
```

Comment:
Is "unmarshal connection" important, or could we manage with "unmarshal json". Helps modifiability aka refactoring in the future...
```go
defer err2.Handle(&err)
```